### DISCUSSION TITLE
can´t upload 3D object

### DISCUSSION THREAD

**Discussion by inst5** [cables-gl/cables_docs#164] [Answers & Questions] [answered]
Hi,
I´ve created a .obj out of a photometry software 3DF Zephyr. But I can´t updaload it to mesh3D.
Also converting it to collada with http://www.greentoken.de/onlineconv/ doesn´t allow me to upload the file.
Is there special thing what I have to consider to puload 3D objects?
thanks!

**Comment by tastez** [comment]
Hi, try converting your OBJ to a GLTF - it is the preferred format currently.

**Comment by inst5** [comment]
Hi,
thanks for that hint. I will try to convert my file and see how it works.

**Comment by inst5** [comment, marked as answer]
Hi,
ok understood now the converting process gltf->glb here and it works. So gltf is a better way to start as obj.
thanks for this tutorial: https://www.youtube.com/watch?v=QNLw21hSweg
